gen3/docs/products/dicom-viewer.md

```diff
@@ -0,0 +1,138 @@
+# DICOM Viewer integration
+
+- [Overview](#overview)
+- [Deployment](#deployment)
+  - [Services](#services)
+  - [Authorization](#authorization)
+  - [Portal explorer integration](#portal-explorer-integration)
+- [Submission](#submission)
+- [Developer notes](#developer-notes)
+
+## Overview
+
+We use the [Ohif viewer](https://ohif.org) for the frontend and the [Orthanc server](https://www.orthanc-server.com) for the backend.
+
+- [Ohif viewer fork](https://github.com/uc-cdis/viewers)
+  - The viewer is accesssible in Gen3 at `<Gen3 portal hostname>/ohif-viewer`, so the `PUBLIC_URL` environment variable must be set to `/ohif-viewer` ([a3f7848](https://github.com/OHIF/Viewers/commit/a3f7848b2f00721a5f4ab994754d828fd00cdfb2)). It can only be set at build time, so the only option was to fork the repository and build the image ourselves ([5c5c48a](https://github.com/OHIF/Viewers/commit/5c5c48ac19e4294c38b8bb03691e1b4250c432ba)).
```

Comment:
```suggestion
  - The viewer is accessible in Gen3 at `<Gen3 portal hostname>/ohif-viewer`, so the `PUBLIC_URL` environment variable must be set to `/ohif-viewer` ([a3f7848](https://github.com/OHIF/Viewers/commit/a3f7848b2f00721a5f4ab994754d828fd00cdfb2)). It can only be set at build time, so the only option was to fork the repository and build the image ourselves ([5c5c48a](https://github.com/OHIF/Viewers/commit/5c5c48ac19e4294c38b8bb03691e1b4250c432ba)).
```